migration/create-b-cases/create-b-cases.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO casesvc."case" (
+  casepk,
+  id,
+  casegroupfk,
+  casegroupid,
+  partyid,
+  sampleunittype,
+  collectioninstrumentid,
+  statefk,
+  actionplanid,
+  createdby,
+  createddatetime,
+  optlockversion
+)
+SELECT
+  nextval('casesvc.caseseq'),
+  gen_random_uuid(),
+  cg.casegrouppk,
+  cg.id,
+  cg.partyid,
+  'B',
+  bicases.collectioninstrumentid,
+  'INACTIONABLE',
+  tempcto.actionplanid,
+  'MIGRATION_SCRIPT',
+  now(),
+  1
+  FROM casesvc.casegroup AS cg
+
+-- Retrieve the collectioninstrumentid from a BI case for each casegroup
+     INNER JOIN (
+       SELECT DISTINCT ON (casegroupfk) casegroupfk, collectioninstrumentid
+         FROM casesvc."case"
+        WHERE sampleunittype = 'BI'
+      ) AS bicases
+     ON cg.casegrouppk = bicases.casegroupfk
+
+-- Get actionplanid from temporary table
+     INNER JOIN (
+       SELECT *
+         FROM casesvc.temp_cto
```

Comment:
This table should be dropped after the script is completed

Reply:
:+1: I'll add it into the README that the temporary table has to be dropped